src/Input/Input.tsx

```diff
@@ -48,18 +56,34 @@ export class Input extends React.PureComponent<Input.Props> {
     const {
       className,
       value,
+      status,
+      wrapper = {},
       ...props
     } = this.props as InputDefaultedProps;
     const {
       _onChange: onChange
     } = this;
 
+    const isSuccess = status === InputStatusT('success');
```

Comment:
why `InputStatusT('...')`? what happened to TS? :P

Reply:
😅 copy-pasta